corehq/apps/sms/handlers/fallback.py

```diff
@@ -33,6 +39,8 @@ def fallback_handler(verified_number, text, msg):
         send_message_to_verified_number(verified_number, domain_obj.default_sms_response,
                                     metadata=outbound_meta)
         outbound_subevent.completed()
+    elif is_connect_message:
```

Comment:
it would be nice to leave a comment about why we have to save this but not other types